### DISCUSSION TITLE
my friend get ban 84h for using this tool on pc

### DISCUSSION THREAD

**Discussion by miloud-elbiad88** [MohamedKVIP/MK-PUBG-Mobile-Tool#5] [General]
my friend get ban 84h for using this tool on pc

**Comment by code-fanatic** [comment]
in pubg mobile global version?

**Comment by MohamedKVIP** [comment]
> my friend get ban 84h for using this tool on pc

Can you send me a screenshot of the ban? Also, could you confirm which version of PUBG Mobile your friend is using? Make sure to download the tool from my GitHub repository. I've been using this tool since 2021 without any issues.

**Reply by mo5tarasran** [reply]
عندي مشكله ان الاداه بتخرج تلقائي اول مدوس علي connect to gameloop والواجهة تختفي ..انا مشغلها ع الاب توب
